tests/SprykerSdkTest/Spryk/Integration/Spryker/Zed/Business/AddCrudFacadeTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdkTest\Spryk\Integration\Spryker\Zed\Business;
+
+use Codeception\Test\Unit;
+use SprykerSdkTest\SprykIntegrationTester;
+
+/**
+ * @group SprykerSdkTest
+ * @group Spryk
+ * @group Integration
+ * @group Spryker
+ * @group Zed
+ * @group Business
+ * @group AddCrudFacadeTest
+ */
+class AddCrudFacadeTest extends Unit
+{
+    protected SprykIntegrationTester $tester;
```

Comment:
Can you add a test method for this, please? Even if it has no assertions. I expect that the code to test the Spryk is present within this task. So everyone who adds one piece to fulfill the whole picture just needs to add asserts.

Otherwise, everyone who adds a piece will add the same test method which will end up in tons of merge conflicts.

Reply:
fixed